src/resilience4clj/circuit_breaker/metrics.clj

```diff
@@ -74,36 +112,5 @@
    (let [{:keys [prefix metric-registry]
           :or {prefix MetricNames/DEFAULT_PREFIX
                metric-registry m/default-registry}} opts]
+     (unregister circuit-breaker-or-registry prefix metric-registry)
```

Comment:
So, no way to just check if a breaker or registry already exists?

Reply:
There may be in the underlying metrics library, but `unregister` will ultimately perform that check with a call to the library. It will search for an object based on a filter [here](https://github.com/tessellator/resilience4clj-metrics/pull/1/files#diff-c722eb3b93187db3abc7138b04ddfc6eR28-R31) (based on a name prefix) and remove the matches. Since `unregister` is idempotent I thought calling it here would be the simplest solution.

Reply:
I thought that maybe unregistering might "undo" more than you might want and there would a lighter weight simpler check. Looks like not. Does unregistering reset metrics? If so, then so does registering. So, just depends on the full semantics of registering twice you want to present. But the '!' does make it clear that there are side-effects. And I'm probably thinking too much.

Reply:
Unregistering does not reset metrics; the metrics are managed in the circuit breaker itself. Registering just exposes those metrics to a Codahale metrics reporter.